### DISCUSSION TITLE
[Mirror site] OneDrive 镜像站

### DISCUSSION THREAD

**Discussion by YerongAI** [lyswhut/lx-music-desktop#1187] [Show and tell]
您好作者大大，感谢您开发了如此优秀的软件。

我在自己的 OneDrive 镜像站上添加了作者大大的软件，设置了每小时自动同步一次 GitHub Release，如果作者大大有需要，可以将此链接作为额外的下载地址呢：

<https://onedrive.officetool.plus/?/Softwares/lx-music-desktop/>
<https://onedrive.officetool.plus/?/Softwares/lx-music-mobile/>

如果有其他任何疑问或者问题，欢迎随时留言哦！

**Comment by lyswhut** [comment]
谢谢，不过[electron-builder](https://www.electron.build/auto-update)提供的GitHub源更新功能似乎不能简单指定备用镜像地址，需要使用自定义更新服务器的形式，等重写更新功能的时候再考虑支持镜像地址的问题 )

**Reply by YerongAI** [reply]
> 谢谢，不过[electron-builder](https://www.electron.build/auto-update)提供的GitHub源更新功能似乎不能简单指定备用镜像地址，需要使用自定义更新服务器的形式，等重写更新功能的时候再考虑支持镜像地址的问题 )

好的，不过好像程序的更新也从来没正常工作过啊，我都是手动下载新版本的 😂
